cmd/prometheus/main.go

```diff
@@ -252,6 +252,9 @@ func main() {
 		"Size at which to split the tsdb WAL segment files. Example: 100MB").
 		Hidden().PlaceHolder("<bytes>").BytesVar(&cfg.tsdb.WALSegmentSize)
 
+	a.Flag("storage.tsdb.disable-isolation", "Disable tsdb isolation.").
```

Comment:
```suggestion
	a.Flag("storage.tsdb.no-isolation", "Disable tsdb isolation.").
```

We might find a better explanation for the flag? or we would need to provide more details in storage.md for end users?

Reply:
I don't think we need a flag at Prometheus level and should be enough to only have this option at TSDB level. Only if we get requests and have valid reasons for Prometheus to not have isolation, we can add the flag later (it's easier to add a flag than remove it).

Reply:
This has been requested by users in the past, so I would prefer to have a flag.

Reply:
After a second though, I agree with @codesome . Let's not have a flag for the moment, isolation has proven to be strong & solid. As you said, we could still have it later if needed.